lib/miq_automation_engine/engine/miq_ae_engine.rb

```diff
@@ -261,6 +261,15 @@ def self.create_automation_attribute_array_value(value)
     end.join(",")
   end
 
+  def self.create_automation_attributes_from_obj_array(objects, attrs_hash)
+    Array.wrap(objects).each do |object|
+      next unless object
+      key   = create_automation_attribute_key(object)
+      value = create_automation_attribute_value(object)
+      attrs_hash[key] = value
```

Comment:
Should we check the key isn't already set? It seems pretty easy to accidentally pass two objects of the same class in the array.